source/plugin/permissions/definitions.rst

```diff
@@ -0,0 +1,183 @@
+===========
+Definitions
+===========
+
+Permission
+==========
+
+A permission is a case-sensitive??? hierarchical string key that is used to determine whether a Subject can do a specific action or not.
+The string is split into separate parts using the dot character.
+Permissions should be structured like this.
+
+.. code-block:: text
+    
+    <PluginID>.<MainGroup>.<Subgroup1>...
+
+Allowed characters "A" - "Z", "a" - "z", "0" - "9", "_", "-" and ".".
+
+Examples
+~~~~~~~~
+
+* ``myPlugin``
+
+Grants **full** access to all plugin's permissions.
+
+* ``myPlugin.commands``
+
+Grants **full** access to all plugin's commands.
+
+* ``myPlugin.commands.teleport``
+
+Grants **full** access to the plugin's teleport command.
+
+* ``myPlugin.commands.teleport.execute``
+
+Only grants the user the permission to execute the command.
+(With this permission alone the user would only be able to teleport himself to others in his current world.)
+
+* ``myPlugin.commands.teleport.others``
+
+Only grants the user the permission to teleport others.
+(He still needs the ``myPlugin.commands.teleport.execute`` to actually execute the command.)
```

Comment:
It's great to have examples, but this gets a bit long - and is bloated by the repetition of this line. Perhaps you can state that it is a prerequisite for the following permissions. However, is it truly necessary to have an .execute permission, or is that just a choice in your (theoretical) implementation?